src/human_aware_rl/ppo/ppo_rllib_test.py

```diff
@@ -251,9 +250,7 @@ def test_ppo_fp_sp_no_phi(self):
             options={"--loglevel": "ERROR"},
         ).result
         # Sanity check (make sure it begins to learn to receive dense reward)
-        self.assertGreaterEqual(
-            results["average_total_reward"], self.min_performance
```

Comment:
Ise `self.min_performance` still assigned to a value somewhere? What value does it have? It might be good to clean that up

Reply:
I changed the performance thresholds for those with relatively large performance differences. Others have pretty small differences so I just kept the old performance threshold = 5 as sanity checks. Self.min_performance is set in the __init__ method.

Reply:
I see, makes sense! Maybe worth adding this as a comment in the code just have it clear why these are hardcoded to be other values